packages/components/src/powered-by/style.scss

```diff
@@ -0,0 +1,30 @@
+@import '@wordpress/base-styles/variables';
+
+.powered-by {
+	display: inline-flex;
+	flex-wrap: wrap;
+	align-items: center;
+	gap: $grid-unit-10;
+	margin: 20px;
```

Comment:
Let's not add default margins to our components, as it limits their reusability

Reply:
Makes sense, I think I just took it from the figma.

I think a lot of uses will be in the footer of a page, they'll likely want the same margin and also to be center aligned.